gradle.properties

```diff
@@ -7,12 +7,15 @@ dirConfig = config
 dirLib = lib
 dirQuality = staticAnalysis
 
-org.gradle.jvmargs=-Xms256m -Xmx2048m
```

Comment:
Why was it necessary to increase the RAM? If its necessary, I'd recommend to just increase it and not leave the old one commented

Reply:
There where issues with PMD and too low memory settings.